### PR TITLE
[functionapp] Fixes #7534 - add runtime parameter for windows apps

### LINKED ISSUE #7534: functionapp create runtime parameter is ignored
**Describe the bug**
The runtime parameter of `functionapp create` is ignored

**To Reproduce**
run 
```
az functionapp create -g myResGroup -s mystorage37462 -n newFunctionApp --os-type Windows --runtime node -c northeurope
```
When the function app is deployed the application setting **FUNCTIONS_WORKER_RUNTIME** is missing and the portal prompts you to set the runtime.
I don't understand why the `--runtime` parameter exists if it doesn't do anything

**Expected behavior**
The function is created with runtime and **FUNCTIONS_WORKER_RUNTIME** set to `node`

**Environment summary**
Cloud Shell and WSL (installed via apt)
azure-cli (2.0.47)

**Additional context**
Nothing

### PR DESCRIPTION
**Pull request by ankitkumarr**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

**Comment by ankitkumarr** [comment]
Will update this PR with a style check.

**Comment by panchagnula** [comment]
@ankitkumarr , is the CI stuck?

**Comment by ankitkumarr** [comment]
@panchagnula, I believe so. I also don't have access to start a re-build.

**Comment by panchagnula** [comment]
@yugangw-msft , can you help us with the CI on this PR. Wanted to know if there is anything we can do to get this CI to green state so that we can continue with the PR? Thank you.

File: src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py [hunk 1]
```diff
@@ -1876,6 +1883,9 @@ def create_function(cmd, resource_group_name, name, storage_account, plan=None,
     site_config.app_settings.append(NameValuePair(name='AzureWebJobsDashboard', value=con_string))
     site_config.app_settings.append(NameValuePair(name='WEBSITE_NODE_DEFAULT_VERSION', value='8.11.1'))
 
+    if runtime:
+        site_config.app_settings.append(NameValuePair(name='FUNCTIONS_WORKER_RUNTIME', value=runtime))
```
Comment (yugangw-msft) [inline src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py]: you can move this  to the new `if` block above, this way you have one less `if`

**Comment by yugangw-msft** [comment]
> Wanted to know if there is anything we can do to get this CI to green state so that we can continue with the PR?

@panchagnula appears a glitch on Travis CI. Next time just close and reopen the PR like i did

**Comment by panchagnula** [comment]
> > Wanted to know if there is anything we can do to get this CI to green state so that we can continue with the PR?
> 
> @panchagnula appears a glitch on Travis CI. Next time just close and reopen the PR like i did

@yugangw-msft , thank you. @ankitkumarr , looks like there are still some fixes to be done here.

File: src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py (hunk 1 above)
Comment (ankitkumarr) [reply src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py]: @yugangw-msft, thanks! I will do that. I didn't to keep the usage check portion separate from the configuration portion.

**Comment by ankitkumarr** [comment]
@panchagnula, there was a weird error where my test recording used an older version of the api. I re-recorded my test and hopefully, it should be fixed. Will wait for the CI build.

**Pull request merged**